src/compose/service-manager.ts

```diff
@@ -310,6 +319,7 @@ export async function start(service: Service) {
 		} catch (e) {
 			// Get the statusCode from the original cause and make sure it's
 			// definitely an int for comparison reasons
+			// QUESTION: does this ever happen?
```

Comment:
You're right & I don't think this happens as these are all Docker status codes. The decode here may be too defensive with little upside

Reply:
We could just remove the decode and just see if something breaks, my guess this was necessary with older engine versions

Reply:
I'm doing the change

Reply:
SGTM @pipex!

Reply:
Done !